packages/forklift-console-plugin/plugin-extensions.ts

```diff
@@ -1,12 +1,15 @@
 import type { EncodedExtension } from '@openshift/dynamic-plugin-sdk';
 import type { NavSection } from '@openshift-console/dynamic-plugin-sdk';
 
+import { extensions as mockConsoleExtensions } from './src/mock-console-extension/dynamic-plugin';
 import { extensions as networkMapExtensions } from './src/modules/NetworkMaps/dynamic-plugin';
 import { extensions as planExtensions } from './src/modules/Plans/dynamic-plugin';
 import { extensions as providerExtensions } from './src/modules/Providers/dynamic-plugin';
 import { extensions as storageMapExtensions } from './src/modules/StorageMaps/dynamic-plugin';
 
 const extensions: EncodedExtension[] = [
+  ...mockConsoleExtensions,
```

Comment:
can you move the isMock test to this file (same with the metadata)?

Reply:
I tried that originally but IMO it looks worse.

Reply:
ok, we will add some gurd that this extention will not happen in production in a follow up,
using this design, the production build will include this extention, even if it's inert it's not nice